archive/f/ferret/README.md

```diff
@@ -0,0 +1,26 @@
+# Sample Programs in Ferret
+
+Welcome to Sample Programs in Ferret!
+
+Ferret takes in Clojure files and compiles them to C++.
+
+## Sample Programs
+
+- [Hello World in Ferret](https://github.com/TheRenegadeCoder/sample-programs/issues/502)
```

Comment:
Can you create a new issue for this line? We usually create a separate issue for the missing article and link it here.

Reply:
I'll do.